maro/simulator/scenarios/finance/account.py

```diff
@@ -0,0 +1,41 @@
+"""Used to maintain stock/futures, one account per episode"""
+from collections import OrderedDict
+
+from maro.backends.frame import NodeAttribute, NodeBase, node
+from maro.simulator.scenarios.finance.common.common import Order, OrderDirection, TradeResult
+
+
+@node("account")
+class Account(NodeBase):
+    remaining_money = NodeAttribute("f")
+    total_money = NodeAttribute("f")
+
+    def __init__(self):
+        self.action_history = OrderedDict()
+
+    def set_init_state(self, init_money: float):
+        self._money = init_money
```

Comment:
align naming

Reply:
Do you have three different kinds of naming for the init_money: _money, init_money, total_money?

Reply:
ok

Reply:
init_cash, remaining_cash, total_assest_value

Reply:
cash asset_value